src/amo/routes.js

```diff
@@ -27,27 +29,33 @@ import ServerError from './components/ErrorPage/ServerError';
 // https://github.com/mozilla-services/puppet-config/tree/master/amo
 
 export default (
-  <Route path="/:lang/:application" component={App}>
-    <IndexRoute component={Home} />
-    <Route path="addon/:slug/" component={Addon} />
-    <Route path="addon/:addonSlug/reviews/" component={AddonReviewList} />
-    <Route path=":visibleAddonType/categories/" component={CategoriesPage} />
-    <Route path=":visibleAddonType/featured/" component={FeaturedAddons} />
-    <Route path=":visibleAddonType/:slug/" component={Category} />
-    <Route path="search/" component={SearchPage} />
-    <Route
-      path="401/"
-      component={config.get('isDevelopment') ? NotAuthorized : NotFound}
-    />
-    <Route path="404/" component={NotFound} />
-    <Route
-      path="500/"
-      component={config.get('isDevelopment') ? ServerError : NotFound}
-    />
-    <Route path="simulate-async-error/" component={SimulateAsyncError} />
-    <Route path="simulate-sync-error/" component={SimulateSyncError} />
-    <Route path="simulate-client-error/" component={SimulateClientError} />
-    <Route path=":visibleAddonType/" component={LandingPage} />
-    <Route path="*" component={NotFound} />
-  </Route>
+  <div>
+    <Route path="/:lang" component={App}>
+      <Route path="about" component={About} />
+      <Route path="review_guide" component={ReviewGuide} />
```

Comment:
Might be nice to TODO: this to have a dash instead. Dashes are cool in URLs and we could do a redirect.

Reply:
I think we should wait on that until after this site has replaced the old one. Changing the urls on only one side is likely to mess things up and it doesn't buy us much given the huge list of other things we need to do.